pkg/machines/xml-helpers/options-helper.js

```diff
@@ -0,0 +1,40 @@
+// This is copied from https://raw.githubusercontent.com/nashwaan/xml-js/master/lib/options-helper.js
+import { isArray } from './array-helper.js';
+
+export function copyOptions(options) {
+    var key; var copy = {};
+    for (key in options) {
+        if (options.hasOwnProperty(key)) {
+            copy[key] = options[key];
+        }
+    }
+    return copy;
+}
```

Comment:
How is `copyOptions(o)` different from `{...o}` (or `Object.assign({}, o)`)?

Reply:
Removed xml-helpers.